src/Raven.Studio/wwwroot/App/views/database/settings/documentsCompression.html

```diff
@@ -0,0 +1,70 @@
+<form role="form" id="documentsCompression" class="flex-vertical absolute-fill content-margin" data-bind="submit: saveChanges" autocomplete="off">
+    <div class="toolbar">
+        <button type="submit" class="btn btn-primary" title="Save Changes" data-bind="enable: isSaveEnabled, css: { 'btn-spinner': spinners.save() }">
+            <i class="icon-save"></i> <span>Save</span>
+        </button>
+    </div>
+    <div class="row">
+        <div class="col-sm-12 col-lg-6">
+            <div class="panel">
+                <div class="padding padding-lg">
+                    <div class="flex-horizontal">
+                        <div class="dropdown btn-group flex-grow">
+                            <input class="form-control dropdown-toggle" placeholder="Select collection (or enter new collection)" data-toggle="dropdown"
+                                   data-bind="textInput: collectionToAdd, attr: { id: 'collectionNameInput' }" />
+                            <span class="caret dropdown-toggle" data-toggle="dropdown"></span>
+                            <ul class="dropdown-menu" role="menu" style="display: none;" data-bind="autoComplete: '#' + 'collectionNameInput', foreach: $root.createCollectionNameAutocompleter()">
+                                <li role="presentation" data-bind="click: _.partial($root.addWithBlink, $data)">
+                                    <a role="menuitem" tabindex="-1" href="#">
+                                        <span data-bind="text: $data"></span>
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                        <button class="btn btn-success" data-bind="click: $root.addCollection, enable: collectionToAdd, attr: { title: collectionToAdd() ? 'Click to add collection to the compressed collections list' : '' }">
+                            <i class="icon-plus"></i>
+                            <span>Add</span>
+                        </button>
+                        <div class=" margin-left margin-left-sm">
+                            <button class="btn btn-info" data-bind="click: $root.addAllCollections, enable: canAddAllCollections, attr: { title: canAddAllCollections() ? 'Add all collections in the database to the compressed collecions list' : '' }">
```

Comment:
I think we don't need logic for title - as this is always the truth

Reply:
done